test/unit/adapters/http.js

```diff
@@ -1078,5 +1080,38 @@ describe('supports http with nodejs', function () {
     });
   });
 
+  it('should allow passing FormData', function (done) {
```

Comment:
👏  Very important test that would have solved recent FormData issues. 
Do you think we should extend this test to include a file?

Reply:
Done. But in fact, we should not test FormData itself. The new version of the `toFormData` helper just appends data fields to the FormData instance, it does not distinguish between string and file data types.
The added nodejs test for the isFormData predicate is the closest test we can do for this purpose. We can't test it in a react-native environment.